mobile/src/core/store/app/actions.ts

```diff
@@ -19,7 +19,8 @@ export const loadApp = createActionFn<void, Promise<void>>(
       dispatch(loadPasscodeEnteredTime()),
       dispatch(loadAppSettings())
     ]);
-    dispatch(loadCMCData());
+    dispatch(loadPriceApiData());
```

Comment:
`cmc` has been refactored to `priceApi` since it now can call coinmarketcap and cryptocompare